### ISSUE TITLE
ETF-LANG example request

### ISSUE THREAD

**Issue by rcgalbo** [sr33j/kalshi-etf-playground#1]
I am not really sure how to get started using this tool. Is there any way that you could add a research example of ETF-LANG?

**Comment by harrodyuan** [comment]
Yeah i think now it require 2 factor authorization and i do not think this workd anymore

**Comment by sr33j** [comment]
Hey sorry! This slipped my mind, but I will work on this issue.

**Comment by harrodyuan** [comment]
yeah actually i think i figured it out in my own Kalshi repo, you can take a look
